### PR TITLE
Changes to rifle rounds

### PR DESCRIPTION
**Pull request by ghost**
Context: http://smf.cataclysmdda.com/index.php?topic=10693.0

Changes:
Renamed 5N7 to 7N10
Changed damage for all 7.62x39mm rounds, all 5.45x39mm rounds, .223, and .308
Changed descriptions for all 7.62x39mm rounds, all 5.45x39mm rounds, .223, and .308
Minor changes to recoil/accuracy/pierce for all affected rounds

### CODE REVIEW

**Comment by chaosvolt** [comment]
Ah, interesting. For once I'm not to blame for any oddities on the 5.45x39mm, as I ported the ammo straight from Icecoon's Weapons Pack.

**Comment by kevingranade** [comment]
Side note, since we're deriving our damage values from real-world stats
now, we need that explained somewhere and either a table of those
real-world stats or a link to them.  Otherwise someone is going to come
along and tweak the values for some reason.

**Comment by chaosvolt** [comment]
That would be good to have references, yes. As I've found when statting out firearms, that can be hard to come by at times.

**Comment by SpongeTheGlaive** [comment]
I'll compile a list of values and relevant information used for each round. Should I include it in a PR or keep it external?

**Comment by kevingranade** [comment]
I think it should go in the doc/ folder in a new file.

**Comment by SpongeTheGlaive** [comment]
Got it. I'll sort that out tomorrow.

File: data/json/items/ammo.json [hunk 1]
```diff
@@ -1598,11 +1598,11 @@
         "to_hit" : 0,
         "ammo_type" : "308",
         "casing" : "308_casing",
-        "damage" : 36,
+        "damage" : 60,
```
Comment (Coolthulhu) [inline data/json/items/ammo.json]: 60 is a lot

**Comment by SpongeTheGlaive** [comment]
Added AMMO_STATS.txt with all relevant info for this PR. I'll update it for the last one when I find the figures.

Semi-OT: Why are the files in ./doc/ all caps?

**Comment by karlnp** [comment]
IIRC documents are written in all-caps historically so they show at the head of a directory listing.

**Comment by chaosvolt** [comment]
Are you plotting a tweak to .50 BMG anytime soon? 10 rounds of AP and API take up the same 2 volume that 30 rounds of Ball do, because someone forgot that stack size is a thing. o3o

**Comment by SpongeTheGlaive** [comment]
@chaosvolt Fixed.

**Comment by chaosvolt** [comment]
...I said volume. >_>

**Comment by SpongeTheGlaive** [comment]
10 .50 rounds now take one volume instead of 0.6 with stack sizes being accounted for. This is reasonable given the size of the ammunition.

**Comment by chaosvolt** [comment]
...hmm. But, are these variants actually bulkier than Ball rounds? >____>

**Comment by SpongeTheGlaive** [comment]
All 3 types have the same volume per round now. Stack size is the only difference.

Different bullets can't be significantly larger than standard ball because they would not fit in .50 cases.

**Comment by chaosvolt** [comment]
As I see. Sorry about that.

**Comment by SpongeTheGlaive** [comment]
That's all the rifle rounds done. This is ready to merge now.

**Comment by chaosvolt** [comment]
Ooh, interesting. I do regret that we can't have the tracer properties show up on an "every few shots" basis, so as it is every round fired from the tracer mix has its incendiary properties.

**Comment by SpongeTheGlaive** [comment]
If I can work out how ranged.cpp works I might try to implement it. I have the c++ skills of a caveman so I doubt it will be done soon, if ever.

**Comment by chaosvolt** [comment]
No idea how to implement such a thing either. I'm struggling with rather basic things like adding vehicle groups to mapgen_functions.cpp.

**Comment by karlnp** [comment]
*adds it to his pile*

**Pull request merged**